website/pages/docs/configuration/service-registration/kubernetes.mdx

```diff
@@ -0,0 +1,148 @@
+---
+layout: docs
+page_title: Kubernetes - Service Registration - Configuration
+sidebar_title: Kubernetes
+description: >-
+  Kubernetes Service Registration labels Vault pods with their current status
+  for use with selectors.
+---
+
+# Kubernetes Service Registration
+
+Kubernetes Service Registration tags Vault pods with their current status for
+use with selectors. Service registration is only available when Vault is running in
+[High Availability mode](/docs/concepts/ha).
+
+- **HashiCorp Supported** – Kubernetes Service Registration is officially supported
+  by HashiCorp.
+## Configuration
+
+```hcl
+service_registration "kubernetes" {
```

Comment:
Should we make a note that `service_registration "kuernetes" {}` is still required if you're using the environment variables?

Reply:
Oh, good idea. Yes.

Reply:
https://github.com/hashicorp/vault/pull/8374/commits/64aafed0712552369192c4121fb8c3308d8308d7